ibm/structures.go

```diff
@@ -722,7 +722,7 @@ func expireRuleGet(in []*s3.LifecycleRule) []interface{} {
 				rule["days"] = int(*(r.Expiration).Days)
 			}
 			if r.Filter != nil {
-				rule["prefix"] = *(r.Filter).Prefix
+				rule["prefix"] = r.Filter.Prefix
```

Comment:
This should n't be this way to handle the nil ...If we delete * then it will stores the value of the address
We have PR from COS team who fixed it to handle it
https://github.com/IBM-Cloud/terraform-provider-ibm/pull/2159

Reply:
ah, ok, I'll test out this other fix #2159  locally and report back here

Reply:
@hkantare I tested #2159 and that worked without issue.  When do you plan to cut a new release with this fix? Thanks.